### PR TITLE
remove unused flags

### PR DESCRIPTION
**Pull request by godrei**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -110,38 +94,24 @@ func executePhases(cmd cobra.Command, progress *phases.Progress) error {
...
-	}
-	if progress.AutoCodesign == nil {
-		codesign, err := phases.AutoCodesign(projectType)
-		if err != nil {
-			return err
-		}
-		_ = codesign
-		progress.AutoCodesign = nil
+	// codesign
+	_, err = phases.AutoCodesign(projectType)
```
Comment (trapacska) [inline cmd/root.go]: Is it in-progress?

File: cmd/root.go [hunk 2]
```diff
@@ -110,38 +94,24 @@ func executePhases(cmd cobra.Command, progress *phases.Progress) error {
...
-		progress.AddWebhook = &cmdFlagAddWebhook
-	}
-	if progress.AddWebhook == nil {
-		wh, err := phases.AddWebhook()
-		if err != nil {
-			return err
-		}
-		progress.AddWebhook = &wh
+	// webhook
+	wh, err := phases.AddWebhook()
```
Comment (trapacska) [inline cmd/root.go]: I think this can be a flag also, a very straightforward boolean flag, wdyt?

File: cmd/root.go [hunk 3]
```diff
@@ -40,52 +32,43 @@ var (
...
 			return err
 		}
-		progress.Public = &public
+		progress.Public = public
 	}
 
-	if cmd.Flags().Changed(cmdFlagKeyRepo) {
-		progress.Repo = &cmdFlagRepo
+	// repo
+	repoDetails, err := phases.Repo(progress.Public)
```
Comment (trapacska) [inline cmd/root.go]: I think this can be a flag also. I would only remove the flags which requires mixed logic to compose. Wdyt?

Comment (godrei) [reply cmd/root.go]: the thing is that the tool needs to run in the directory of the project, so if the repo url is given, we need another set of info/actions still

File: cmd/root.go (hunk 2 above)
Comment (godrei) [reply cmd/root.go]: that is true, but for now i would like to focus on the requirements

File: cmd/root.go (hunk 1 above)
Comment (godrei) [reply cmd/root.go]: yep, most likely the ios codesigning files will not be returned, but codesigndoc will have an option to auto upload them

**Pull request merged**